acceptance/repository-cloudant/setup.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash		
+ 		
+ # Shell script to start the database and app services before running the tests.		
+ 		
+ ## color codes		
+ RED='\033[1;31m'		
+ GREEN='\033[1;32m'		
+ YELLOW='\033[1;33m'		
+ CYAN='\033[1;36m'		
+ PLAIN='\033[0m'		
+ 		
+ ## variables		
+ COUCHDB_CONTAINER="couchdb3"
+ HOST=localhost		
+ USER='admin'		
+ PASSWORD='pass'		
+ PORT=5984		
+ DATABASE='testdb'				
+ if [ "$1" ]; then		
+     HOST=$1		
+ fi		
+ if [ "$2" ]; then		
+     PORT=$2		
+ fi		
+ if [ "$3" ]; then		
+     USER=$3		
+ fi		
+ if [ "$4" ]; then		
+     PASSWORD=$4		
+ fi		
+ if [ "$5" ]; then		
+     DATABASE=$5		
+ fi			
```

Comment:
The positional parameters seem to be error prone.

Reply:
Could you explain more? Cause we have such setups for other connectors such as MySQL Postgres. Thanks!

Reply:
If other scripts use the same convention, I can live with it. Ideally, the order of these params should not matter. `--user my-user --db my-db` style is preferred.